apps/protocol-api/src/main.ts

```diff
@@ -105,6 +105,7 @@ const permissions = shield(
       createUserAttestation: and(ownsData, isAuthenticated),
       createUserContribution: and(ownsData, isAuthenticated),
       createUserCustom: or(hasToken, and(ownsData, isAuthenticated)),
+      createGuildUserCustom: or(hasToken, and(ownsData, isAuthenticated)),
```

Comment:
Can we remove both has token and ownsData here? A job isn't calling this and ownsData is essentially allow